common/modal.js

```diff
@@ -84,6 +87,10 @@
             $uibModalInstance.close("reload");
 
         };
+        vm.login = function () {
+            $uibModalInstance.close("login");
```

Comment:
Based on changes that I made in a PR 2 days ago regarding the 2 modals overlapping each other, we decided that "login" links should not close the current modal but instead leave the modal open and display the login window.

This line should be:
`Session.loginInAPopUp();`